mobilecoind/src/service.rs

```diff
@@ -2699,6 +2700,11 @@ macro_rules! build_api {
                     request: $service_request_type,
                     sink: UnarySink<$service_response_type>,
                 ) {
+                    let tracer = mc_util_telemetry::tracer!();
```

Comment:
This ends up creating a span for every mobilecoind function, but it will only be for those that are run with MC_TELEMETRY=1 so figure it's ok and shouldn't be too spammy

Reply:
See #4094 i've reworked this, as I was wrong, it's a bit spammy